keras/utils/vis_utils.py

```diff
@@ -290,7 +290,7 @@ def format_shape(shape):
             [format_shape(ishape) for ishape in layer.input_shapes])
       else:
         inputlabels = '?'
-      label = '{%s}|{input:|output:}|{{%s}}|{{%s}}' % (label, inputlabels,
+      label = '{%s}|{input:|output:}|{{%s}|{%s}}' % (label, inputlabels,
                                                        outputlabels)
```

Comment:
Please fix indent here

Reply:
I understand you mean the EOL after inputlables, I fixed it:

![image](https://user-images.githubusercontent.com/11573780/155269859-e3e17b7f-f059-44ee-85bf-0bde4b42161b.png)